vep/__init__.py

```diff
@@ -6,8 +6,12 @@
 import os
 from ConfigParser import RawConfigParser
 import re
+import shutil
 import sys
 
+__version__ = '0.0.14'
```

Comment:
nit: 2 spaces between top-level declaration blocks. The two newlines after it seem superfluous, because you are still declaring module-level variables.